src/sql/workbench/parts/notebook/notebook.component.ts

```diff
@@ -441,7 +441,7 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 	protected initNavSection(): void {
 		this._navProvider = this.notebookService.getNavigationProvider(this._notebookParams.notebookUri);
 
-		if (this.contextKeyService.getContextKeyValue('isDevelopment') &&
+		if (this.contextKeyService.getContextKeyValue('notebookQuality') !== 'stable' &&
```

Comment:
minor: you could just check `product.quality` here and verify it does not equal stable.